pol-core/bscript/bobject.h

```diff
@@ -116,6 +116,7 @@ class BObjectImp : public ref_counted
     OTFuncRef = 39,
     OTExportScript = 40,
     OTStorageArea = 41,
+    OTContinuation = 42,
```

Comment:
Since it's only internally used and has no script representation would be better to leave don't know 100 gap and add a comment that's it's internal. Otherwise we will wonder after a few weeks and when we add a new script object the scripters will wonder where TypeOfInt 42 is